cogs/vxtwitconverter/vxtwitconverter.py

```diff
@@ -0,0 +1,44 @@
+from redbot.core import commands
+from redbot.core.bot import Red
+from discord import Message
+from urlextract import URLExtract
```

Comment:
Is this a third party module? If so, please create a `info.json` and specify this as a requirement.